packages/shared-data/json/supported-models.json

```diff
@@ -1,27 +1,106 @@
 [
   {
-    "name": "meta-llama/llama-4-scout-17b-16e-instruct",
-    "displayName": "LLaMA 4 Scout",
-    "rateLimit": 40000000
+    "name": "openai/gpt-oss-120b",
+    "displayName": "OpenAI GPT OSS 120B",
+    "rateLimit": 400000000,
+    "providers": [
+      {
+        "bedrock": {
+          "id": "openai.gpt-oss-120b-1:0"
```

Comment:
Wondering whether we need that additional sub-object with "id" property or can just assign the value to the provider name directly? Do we anticipate additional properties in the future?